packages/react/src/drawer/docs/overview.mdx

```diff
@@ -39,13 +39,17 @@ relatedPatterns: ['search-filters']
 
 <DoHeading />
 
+- consider using a side action page first - side action pages support more use cases than side drawers
+- only use side drawers for filters, small overflow form actions and related guidance content if you can’t provide these interactions on the parent page
+- only use side drawers for external facing forms when you need to maintain the user’s context of the parent page
 - read the [Search filters pattern](/patterns/search-filters)
 - use a [Button group](/components/button#button-group) to display actions
 - include a meaningful title - e.g. ‘Filter by’
 - optionally close the Drawer when an action button is pressed
 
 <DontHeading />
 
+- use for long forms or any content that is likely to cause a user to scroll vertically or horizontally in the drawer - use a side action page for these interactions instead
 - remove the ‘Close’ button
```

Comment:
```suggestion
- provide navigation or multiple pages inside a drawer
- remove the ‘Close’ button
```

Reply:
"provide navigation" is a bit vague. Because we do allow put links to content pages afaik (like in the help context feature Pete worked on)